components/create-collaboration/CreateCollaborationMap.tsx

```
@@ -0,0 +1,112 @@
+import { useEffect, useState } from "react";
+import { Platform } from "react-native";
+import MapView, { Marker } from "react-native-maps";
+import { APIProvider } from "@vis.gl/react-google-maps";
+
+import Map from "@/components/map";
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)